### PR TITLE
Adds the Cleft as a start choice at character creation

### PR DESCRIPTION
**Pull request by Hazado**
Added in the option for the Cleft start from Uru:CC

image

### CODE REVIEW

**Comment by dpogue** [comment]
I realize the text has been workshopped to death in Discord, but a new player won't know what "Cleft" or "Relto" mean.
I would swap the order of the words: "New Explorer - Cleft" and "Returning Explorer - Relto".
(Just a comment, not intended to hold up further work/review on this)

**Comment by Hazado** [comment]
> I realize the text has been workshopped to death in Discord, but a new player won't know what "Cleft" or "Relto" mean. I would swap the order of the words: "New Explorer - Cleft" and "Returning Explorer - Relto". (Just a comment, not intended to hold up further work/review on this)

Seems like a good wording improvement to me! Ill get this updated shortly.

File: compiled/dat/GUIEnglish.loc [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<localizations>
+	<age name="GUI">
+        <set name="Dialog04b">
+			<element name="CreateExplorer">
+				<translation language="English">CREATE EXPLORER</translation>
+			</element>
+		</set>
```
Comment (Hoikas) [inline compiled/dat/GUIEnglish.loc:8]: It looks like there's a mixture of tabs and spaces here. The indentation looks funny on GitHub.

File: compiled/dat/GUIEnglish.loc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		<set name="Dialog04d">
+			<element name="InvalidName">
+				<translation language="English">Error: invalid Name. Please enter another.</translation>
```
Comment (Hoikas) [inline compiled/dat/GUIEnglish.loc:11]:
```suggestion
				<translation language="English">Error: Invalid Name. Please enter another.</translation>
```

File: compiled/dat/GUIEnglish.loc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+			</element>
+			<element name="ErrorName">
+				<translation language="English">Error: you must enter a Name.</translation>
```
Comment (Hoikas) [inline compiled/dat/GUIEnglish.loc:14]:
```suggestion
				<translation language="English">Error: You must enter a Name.</translation>
```

File: compiled/dat/GUIEnglish.loc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+			</element>
+			<element name="ErrorGender">
+				<translation language="English">Error: you must select a gender.</translation>
```
Comment (Hoikas) [inline compiled/dat/GUIEnglish.loc:17]:
```suggestion
				<translation language="English">Error: You must select a gender.</translation>
```

File: compiled/dat/GUIEnglish.loc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+			</element>
+			<element name="ErrorPath">
+				<translation language="English">Error: you must choose a Path.</translation>
```
Comment (Hoikas) [inline compiled/dat/GUIEnglish.loc:20]:
```suggestion
				<translation language="English">Error: You must select a Path.</translation>
```

File: compiled/dat/GUIEnglish.loc [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+			</element>
+			<element name="InvalidCharacters">
+				<translation language="English">Warning: you entered invalid characters in your player name.  The invalid characters have been removed, please make sure your player name is still what you want.</translation>
```
Comment (Hoikas) [inline compiled/dat/GUIEnglish.loc:23]:
```suggestion
				<translation language="English">Warning: You entered invalid characters in your player name. The invalid characters have been removed. Please make sure your player name is still what you want.</translation>
```

File: compiled/dat/GUIEnglish.loc [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+			</element>
+			<element name="IncorrectFormatting">
+				<translation language="English">Warning: your player name has some incorrect formatting.  The formatting has been corrected, please make sure your player name is still what you want.</translation>
```
Comment (Hoikas) [inline compiled/dat/GUIEnglish.loc:26]:
```suggestion
				<translation language="English">Warning: Your player name has some incorrect formatting. The formatting has been corrected. Please make sure your player name is still what you want.</translation>
```

File: compiled/dat/GUIEnglish.loc [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+			</element>
+            <element name="NameAlreadyExists">
+				<translation language="English">Error: this player name already exists.</translation>
```
Comment (Hoikas) [inline compiled/dat/GUIEnglish.loc:29]:
```suggestion
				<translation language="English">Error: This player name already exists.</translation>
```

Also funny indentation hereabouts.

**Comment by Hazado** [comment]
Added Help buttons to the creation screen
image
image
image

**Comment by Hazado** [comment]
New Orientation Screen for Cleft Start
image

**Comment by Deledrius** [comment]
Great work on these!

This is a minor suggestion, but I wonder if it might make more sense to reverse the order of the two bits of information.  Especially for new players, the relevant explanation might be better coming first.  That is:

```
For new players who have not played before.
You start in [...]
```

and

```
For returning players familiar with URU.
You start on [...]
```

That way it's tl;dr, then the additional explanation.

> image image

**Comment by Deledrius** [comment]
There is something important missing, but I'm not sure where this should go.  When being faced with a choice at account creation, it's **crucially** important to explain to the users that this choice is _not_ going to be locking them out of content!  There should be a note somewhere (perhaps on both `?` screens for each path) that players will still be able to access the whole game, it's just skipping them ahead and not a permanent thing.

Something along the lines of "This choice starts with a tutorial area." for the new players and "This choice starts after the tutorial area, but it can be visited at any time." for returning players.

Even better might be to incorporate this into the existing text more directly.

> You start in the desert near a dormant volcano.  This area serves as the game's tutorial.

> You start on your personal Relto island.  This skips the game's tutorial, but it can still be visited through the course of normal play.

If/when a fully alternate path is added to the game (@DoobesURU ?), this would need to be modified to explain that (though hopefully that will be accomplished in-game as Uru's fashion is to keep IC as IC as possible).

**Comment by Hoikas** [comment]
I don't remember if the orientation GUI is in xKIGUI.max, but if it is, remember that we need the max file to be updated to ensure that we can continue to make adjustments in 3ds Max.

**Comment by Hazado** [comment]
> I don't remember if the orientation GUI is in xKIGUI.max, but if it is, remember that we need the max file to be updated to ensure that we can continue to make adjustments in 3ds Max.

it is, my version of max is borky at times. ill work to get it the same as my manually edited file

**Comment by Hazado** [comment]
Enlarged the error popup screen, which is what the help text was using.
Added your suggestions @Deledrius 
image
image

**Comment by Hoikas** [comment]
IIRC the current blockage on this is the updating of xKIGUI.max

**Comment by Hoikas** [comment]
It appears that there are multiple GUI localization files whose contents are copies of GUIEnglish.loc. These should be deleted - they will be generated afresh by plLocalizationEditor when a translator enters translations for these items.

**Comment by Hoikas** [comment]
I've rebased this PR and exported the contributed max file.

**Pull request merged**